common/test/acceptance/pages/lms/programs.py

```diff
@@ -20,3 +20,11 @@ def are_cards_present(self):
     def is_sidebar_present(self):
         """Check whether sidebar is present."""
         return self.q(css='.sidebar').present
+
+
+class ProgramDetailsPage(PageObject):
+    """Program details page."""
+    url = BASE_URL + '/dashboard/programs/123'
```

Comment:
Do we even need this fake URL to pass the acceptance test? If not, please remove.